src/WebUI/Pods/PodContentReader.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+    Copyright (c) Microsoft Corporation. All rights reserved.
+    Licensed under the MIT license.
+*/
+
+import { BaseComponent, css } from "@uifabric/utilities";
+import { CardContent, CustomCard } from "azure-devops-ui/Card";
+import * as React from "react";
+import { IVssComponentProperties } from "../Types";
+
+// basic editor functionality
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
+// languages supported in editor
+import "monaco-editor/esm/vs/basic-languages/yaml/yaml.contribution";
+// additional functionality of editor
+import "monaco-editor/esm/vs/editor/browser/controller/coreCommands";
+import "monaco-editor/esm/vs/editor/contrib/find/findController";
+
+export interface IReaderProps extends IVssComponentProperties {
+    text: string;
+    options?: monaco.editor.IEditorConstructionOptions;
+    contentClassName?: string;
+}
+
+export class PodContentReader extends BaseComponent<IReaderProps> {
+    public render(): JSX.Element {
+        return (
+            <CustomCard className={css(this.props.className || "", "k8s-card-padding", "flex-grow")}>
+                <CardContent className={css(this.props.contentClassName || "", "reader-content")} contentPadding={false}>
+                    <div className="reader-outer" style={{ width: "100%", height: "500px", position: "relative" }}>

```

Comment:
i am hard coding the hegith of the reader as 500px, could not find a way to compute dynamically, will do later